src/containers/flows/CancelBid/hooks/useCancelBid.ts

```diff
@@ -52,9 +56,10 @@ const useCancelBid = (): UseCancelBidReturn => {
 					cancelBidOnChain,
 					library.getSigner(),
 				);
-			} catch (e) {
+			} catch (e: any) {
 				console.error(e);
-				throw new Error(constants.ERRORS.SIGNATURE);
+				const errorText = getDisplayErrorMessage(e.message);
+				throw new Error(errorText);
```

Comment:
nit: I generally like to just do `throw new Error(getDisplayErrorMessage(e.message))` rather than assign a variable and then do a single operation with it.

Reply:
Yeah sounds good to me, not sure why but I thought that was the practice to follow but never made sense to me having the additional line, unless being reused. Thanks. Updated

Reply:
I think it makes sense when it's not clear what the variable is - for function calls with one parameter, or when using functions/variables with already explicit names, it's good practice.